.github/workflows/lint.yml

```diff
@@ -0,0 +1,41 @@
+name: golangci-lint
+on: [pull_request]
+permissions:
+  contents: read
+  # Optional: allow read access to pull request. Use with `only-new-issues` option.
```

Comment:
does this mean, we have to set `only-new-issues: true` in line 31?

Reply:
no, we do not want `only-new-issues`. Its the same in nats-manager [here](https://github.com/kyma-project/nats-manager/blob/main/.github/workflows/lint.yml)